rmgpy/data/kinetics/common.py

```diff
@@ -90,10 +90,6 @@ def sortEfficiencies(efficiencies0):
         #Write out additional data if depository or library
         #kinetic rules would have a Group object for its reactants instead of Species
         if isinstance(entry.item.reactants[0], Species):
-            if entry.label != str(entry.item):
```

Comment:
I guess I don't quite understand how reading the entries work if the reaction label does not have the names of species inside it.